ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -16,80 +16,90 @@
+      recordList.add(new ConsumerRecord<CommandId, Command>("T", 1, 1, (CommandId) commandPair
+          .getLeft(), (Command) commandPair.getRight()));
+    }
+    Map<TopicPartition, List<ConsumerRecord<CommandId, Command>>> recordMap = new HashMap<>();
+    recordMap.put(new TopicPartition("T", 1), recordList);
+    CommandStore commandStore = mock(CommandStore.class);
+    expect(commandStore.getNewCommands()).andReturn(new ConsumerRecords<>(recordMap));
+    expect(commandStore.getPriorCommands()).andReturn(Collections.emptyList());
+    replay(commandStore);
+    return new CommandRunner(statementExecutor, commandStore);
+  }
 
-    Map<String, Object> commandConsumerProperties = new HashMap<>();
-    commandConsumerProperties.put("bootstrap.servers", "localhost:9092");
-    Serializer<Command> commandSerializer = new KafkaJsonSerializer<>();
-    Deserializer<Command> commandDeserializer = new KafkaJsonDeserializer<>();
-    Serializer<CommandId> commandIdSerializer = new KafkaJsonSerializer<>();
-    Deserializer<CommandId> commandIdDeserializer = new KafkaJsonDeserializer<>();
+  private StatementExecutor getMockStatementExecutor() throws Exception {
+    CommandId topicCommandId =  new CommandId(CommandId.Type.TOPIC, "_CSASTopicGen");
+    CommandId csCommandId =  new CommandId(CommandId.Type.STREAM, "_CSASStreamGen");
+    CommandId csasCommandId =  new CommandId(CommandId.Type.STREAM, "_CSASGen");
+    CommandId ctasCommandId =  new CommandId(CommandId.Type.TABLE, "_CTASGen");
+    Map<CommandId, CommandStatus> statusStore = new HashMap<>();
+    statusStore.put(topicCommandId, new CommandStatus(CommandStatus.Status.SUCCESS, "Success"));
+    statusStore.put(csCommandId, new CommandStatus(CommandStatus.Status.SUCCESS, "Success"));
+    statusStore.put(csasCommandId, new CommandStatus(CommandStatus.Status.SUCCESS, "Success"));
+    statusStore.put(ctasCommandId, new CommandStatus(CommandStatus.Status.ERROR, "Error"));
 
-    KafkaConsumer<CommandId, Command> commandConsumer = new KafkaConsumer<>(
-        commandConsumerProperties,
-        commandIdDeserializer,
-        commandDeserializer
-    );
+    StatementExecutor statementExecutor = mock(StatementExecutor.class);
+    expect(statementExecutor.getStatuses()).andReturn(statusStore);
 
-    CommandRunner commandRunner = new CommandRunner(statementExecutor, new MockCommandStore
-        ("CT", commandConsumer, null,
-         new CommandIdAssigner(new MetaStoreImpl())));
-    return commandRunner;
+    return statementExecutor;
   }
 
-
   @Test
-  public void testThread() throws InterruptedException {
-    MockKsqkEngine mockKsqkEngine = new MockKsqkEngine(
-        TestUtils.getMockKsqlConfig(), new MockKafkaTopicClient());
-    StatementParser statementParser = new StatementParser(mockKsqkEngine);
-    StatementExecutor statementExecutor = new StatementExecutor(mockKsqkEngine, statementParser);
+  public void testNewCommandRun() throws Exception {
```

Comment:
what does the test do? It is easier to understand what should happen if we have good test names, i.e, "shouldExecuteNewCommandsFromCommandStore"

Reply:
renamed the tests.